src/ui-kit/components/pagination/pagination.spec.ts

```diff
@@ -36,7 +36,7 @@ describe('The Sam Pagination component', () => {
 
     const ellipsis = fixture.debugElement
       .queryAll(By.css('.usa-pagination li span'));
-    const expectedLen2 = 11;
+    const expectedLen2 = 2;
```

Comment:
The extra span tags were removed, hence the change in this unit test that queries for span tags to find ellipses.